packages/shared/src/hooks/notifications/useBookmarkReminder.ts

```diff
@@ -59,14 +62,15 @@ export const getRemindAt = (
   }
 };
 
-export const useBookmarkReminder = (): UseBookmarkReminder => {
+export const useBookmarkReminder = (post: Post): UseBookmarkReminder => {
```

Comment:
Let's do object syntax `{ post: Post }` as with other hooks.

Reply:
Oh, yep. Totally forgot about that.

Reply:
Yeah, easier to add additional props if we need to 👌